src/main/java/mate/academy/spring/validation/RepeatPassword.java

```diff
@@ -0,0 +1,19 @@
+package mate.academy.spring.validation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+@Constraint(validatedBy = RepeatPasswordValidator.class)
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface RepeatPassword {
```

Comment:
you don't use this annotation in your code, so remove it or use it where it's appropriate